### DISCUSSION TITLE
Use SAP UI 5 with angular

### DISCUSSION THREAD

**Discussion by vavasilev** [UI5/webcomponents#5718] [Q&A] [answered]
Hello Colleagues,

I am part of iLM Sink team and we have SAP UI 5 UI. We are considering currently transition to framework, preferably Angular. In this relation I have some questions:
1. What is the difference between the web components project: https://sap.github.io/ui5-webcomponents/ and the fundamental library: https://sap.github.io/fundamental-ngx/#/core/home? 
2. In our application we have lot of graphical dashboards that use lot of stuff from “sap.viz” namespace. As far as what I was able to find in google, I didn’t find anything we can use with Angular - only basic components. For react we found some libraries here: https://sap.github.io/ui5-webcomponents-react/?path=/docs/charts-linechart--default-story. So, what is your recommendation to use for graphics in an Angular project? 

Thanks, Vasil

**Comment by ilhan007** [comment, marked as answer]
Hi @vavasilev 
**UI5 Web Components** https://sap.github.io/ui5-webcomponents/ 
is a web components based library that works in **Angular, React, Vue and whatever web framework comes in future** as it's based on web standards, natively supported. It's developed and supported by the UI5 team, the same developers working on SAPUI5/OpenUI5
**UI5 Web Components is at 1.6.0 stable version** .

**UI5 Web Components For React** https://sap.github.io/ui5-webcomponents-react/?path=/docs/charts-linechart--default-story
is React implementation of UI5 Web Components. **It provides React wrappers for all the UI5 Web Components**, adds more components (as charts, Object Page, Analytical Table), and improves the DEV experience as React has some limitation when it comes to event binding, boolean props binding to web components in general. It's driven by separate team, that we (from UI5 Web Components) are working with. UI5 Web Components For React are to release stable version until the end of the year. **Currently it's beta 0.27.3.**

**Fundamental NGX**  https://sap.github.io/fundamental-ngx/#/core/home  is another project that started separately and **it provides Angular components that can be used in Angular only**.
Just recently it became part of the UI5 org and we are in discussions how to join forces. For example one of the ideas is to reuse the UI5 Web Components internally, e.g. Fundamental NGX Angular components to wrap the UI5 Web Components (same way as UI5 Web Components For React). And, I**t's beta as well 0.35.4.**

We don't have charts components for Angular as of now. We might consider extending the components set with several charts, but it's not something that you can expect short or mid-term. So, now I would suggest using some other charts library.

**Reply by vavasilev** [reply]
Thanks for the detailed response!
